test/vault-api/Skyflow.test.js

```diff
@@ -1930,6 +1941,78 @@ describe('get method with options', () => {
     });
 
 
+  });
+
+  test('get method should send request url with single column name for multiple column value', (done) => {
+
+    let reqArg;
+    const clientReq = jest.fn((arg) => {
+      reqArg = arg;
+      return Promise.resolve({data:getByIdRes})
+    });
+
+    const mockClient = {
+      config: skyflowConfig,
+      request: clientReq,
+      metadata: {}
+    }
+
+    clientModule.mockImplementation(() => { return mockClient });
+    skyflow = Skyflow.init({
+      vaultID: '<VaultID>',
+      vaultURL: 'https://www.vaulturl.com',
+      getBearerToken: () => {
+        return new Promise((resolve, _) => {
+          resolve("token")
+        })
+      }
+    });
+
+    const response = skyflow.get(getByIdWithValidMultipleUniqColumnOptions);
+    response.then((res) => {
+      expect((reqArg.url).match(/column_name=abc/gi)?.length).toBe(1);
+      done();
+    }).catch((er) => {
+      done(er)
+    });
+
+
+  });
+
+  test('get method should send request url with single column name for multiple column value', (done) => {
+
+    let reqArg;
+    const clientReq = jest.fn((arg) => {
+      reqArg = arg;
+      return Promise.resolve({data:getByIdRes})
+    });
+
+    const mockClient = {
+      config: skyflowConfig,
+      request: clientReq,
+      metadata: {}
+    }
+
+    clientModule.mockImplementation(() => { return mockClient });
+    skyflow = Skyflow.init({
+      vaultID: '<VaultID>',
+      vaultURL: 'https://www.vaulturl.com',
+      getBearerToken: () => {
+        return new Promise((resolve, _) => {
+          resolve("token")
+        })
+      }
+    });
+
+    const response = skyflow.get(getByIdWithValidMultipleUniqColumnOptions,{encodeURI:false});
```

Comment:
@roshmik  `{encode: false}` is the option we have added in this test and not was in the above test.
My bad, didn't add it in description.